### DISCUSSION TITLE
Two keys, one key include og plots and nft plots, one key only include og plots

### DISCUSSION THREAD

**Discussion by followgithub** [Chia-Network/chia-blockchain#8213] [Support]
Two keys on one machine, one key includes og plots and nft plots, all plots are about 1Pib .
 One key only include og plots and not use pool， all plots are about 11Pib.
Use 13 harversters pc connetct to the two key farmer(one pc).
All logs show work fine, farmer send new_signage_point_harvester to every peer, get farming_info from peer. Peer log xx plots were eligible for farming .. Found 0 proofs. Everything is normal, but no earn five day. I calculated that this chance is only 0.058%
I use chia1.2.3. Can the version support this two keys setup？

**Comment by loppefaaret** [comment]
yes, version 1.2.3 should be fine for handling a mix of OG plots and plotNFT's mixed onto different keys

**Reply by followgithub** [reply]
From the log, it supports mixed mode.
But I can’t wait any longer. I used two servers to run different keys farmer yesterday afternoon , now I win three block.

**Comment by BrandtH22** [comment]
@followgithub , the newest version of chia has fixes in place to resolve this issue. Please download it from here: https://www.chia.net/downloads/

Since this issue has been resolved with the latest version(s) of chia we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
